### ISSUE TITLE
Feature request: Migrate to null safety [flutter 2.0]

### ISSUE THREAD

**Issue by wmla419** [ArcticZeroo/flutter-search-bar#42]
I really like your package and I hope you can update so that it will support flutter 2.0

**Comment by urmatt** [comment]
For now you can use this:
`flutter_search_bar:
    git:
      url: git://github.com/urmatt/flutter-search-bar.git
      path:`

**Comment by urmatt** [comment]
For now you can use this:
```
flutter_search_bar: 
    git: 
      url: git://github.com/urmatt/flutter-search-bar.git 
      path:
```

**Comment by ArcticZeroo** [comment]
This will be closed as of #46 . Sorry for the delay, finally finished the semester (:

**Comment by ArcticZeroo** [comment]
This is now live with `3.0.0-dev.1` (dart recommends we publish as a pre-release first)
